<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QWidget" name="widget_6" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>成品宽</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditWidth">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>成品长</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditHeight">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <family>黑体</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <bold>false</bold>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="text">
               <string>mm</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="pushButtonLoad">
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>加载文件</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>生成GCODE</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="font">
                  <font>
                   <family>黑体</family>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>阈值：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="thresholdNum">
                 <property name="font">
                  <font>
                   <family>黑体</family>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="sliderThreshold">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QPushButton" name="pushButtonSureImage">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonReset">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>重置</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>文件名：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelFileName">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>文件大小：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelFileSize">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>图片尺寸：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelImageWH">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>unknown</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>250</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonExit">
            <property name="font">
             <font>
              <family>黑体</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>预览</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="imgshowzoom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>500</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string>请选择图片</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
